ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml

```diff
@@ -170,6 +170,7 @@ tests:
     dependencies:
       OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:arm64-latest
     env:
+      AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
```

Comment:
Shall we also update the job name with `mini-perm` for better filtering?

Reply:
My thought is to enable `AWS_INSTALL_USE_MINIMAL_PERMISSIONS` on all 4.18 AWS IPI jobs after this PR get merged

Reply:
It's a good idea, just one little concern is if the minimal permission testing is blocked, then we can't run any 4.18 AWS CI.

Reply:
Oh, you're right, how about select 50% jobs to enable mini-perm

Reply:
@yunjiang29 it's ok to me